### PR TITLE
Chore fixy refactor

### PR DESCRIPTION
**Pull request by DanEvans485**
10/10 would add labels again

### CODE REVIEW

File: app/Traits/ItemsTrait.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+use App\Models\Items\Item;
+
+trait ItemsTrait
+{
+    public static function addItem(
+        Item $item,
+        int $quantity = 1
+    ) {
+        dd('Add Item', $item, $quantity);
```
Comment (Newobc) [inline app/Traits/ItemsTrait.php]: Intentional?

File: app/Http/Resources/ActionResource.php [hunk 2]
```diff
@@ -11,9 +11,8 @@ public function toArray($request)
         return [
             'name' => $this->name,
             'type' => $this->type,
-            'dice' => $this->dice->sides,
-            'dice_count' => $this->dice_count,
-            'as_string' => $this->dice_as_string()
+//            'dice' => $this->dice->sides,
+//            'dice_count' => $this->dice_count,
```
Comment (G-Bro) [inline app/Http/Resources/ActionResource.php]: why?

File: app/Traits/ItemsTrait.php (hunk 1 above)
Comment (DanEvans485) [reply app/Traits/ItemsTrait.php]: Yes not finished yet

File: app/Http/Resources/ActionResource.php (hunk 2 above)
Comment (DanEvans485) [reply app/Http/Resources/ActionResource.php]: Broke need to fix, but later

**Pull request merged**